test_files/type_and_value/type_and_value.js

```diff
@@ -7,13 +7,14 @@ goog.module('test_files.type_and_value.type_and_value');var module = module || {
  */
 
 var conflict = goog.require('test_files.type_and_value.module');
+const tsickle_forward_declare_1 = goog.forwardDeclare("test_files.type_and_value.module");
```

Comment:
I wonder if the relative order of goog.require / goog.forwardDeclare matters.

Reply:
I don't think so. Fingers crossed.